code.test.js

```diff
@@ -7,7 +7,7 @@ const test =
     jsc.forall("array nat", function(arr) {
         var a1 = JSON.parse(JSON.stringify(arr));
         var a2 = JSON.parse(JSON.stringify(arr));
-        var count = permutationSort(a1);
+        var count = permutationSort(a1,0);
```

Comment:
Please don't change the prototype of the function. You could have an internal wrapper function.